requirements-dev.txt

```diff
@@ -0,0 +1,6 @@
+Pillow==4.0.0
```

Comment:
There's already a requirements-test.txt file that is serving the same purpose, can you update that one and remove this?